pantry-finder-api/config/routes.rb

```diff
@@ -4,6 +4,9 @@
   namespace :api do
     resources :agencies, only: :index
   end
+  namespace :api do
+    resources :foodbanks, only: :index
+  end
```

Comment:
No need to make a second `namespace :api` block here. You can simply add your resource on line 6 within the existing api namespace block.

Reply:
thanks for the tip!